config/deploy.rb

```diff
@@ -7,7 +7,7 @@
 # Default deploy_to directory is /var/www/my_app_name
 set :deploy_to, "/opt/app/mirador/#{fetch(:application)}"
 
-set :linked_dirs, %w(log tmp/pids tmp/cache tmp/sockets vendor/bundle config/certs config/settings)
+set :linked_dirs, %w(log tmp/pids tmp/cache tmp/sockets vendor/bundle config/certs config/settings public/assets)
```

Comment:
This seems off to me.  I don't think I have ever had to call out `public/assets` as a linked directory (nor ignored it in my `.gitignore`).

What are we trying to accomplish by doing this?